Saferoom changes.

HeyBanditoz opened the pull request:
Saferooms now behave a little differently.
The bolt buttons have been moved inside of the saferooms.
An impending delamination, radiation storm or red alert will unbolt saferooms.
Saferooms can also be bolted or unbolted via the keycard authentiation device.
Everyone can now operate the bolts inside.

SamSamsonthe3rd commented:
"Sarah crawled to the saferoom door, its shining veneer promising safety and respite from the horrors outside. She reached up, and pushed the access button with a bleeding, broken hand. But the door did not open. It merely flashed its glowing red lights in her face, as if to mock her. A single tear crawled down her cheek as she looked to her right. A tall man in a blood red voidsuit came towards her. He loaded another magazine into his SMG. And as he lifted his gun up and pointed it at her, Sarah whispered to the nothingness,
'Please merge...'"

PurpleMartinJCK commented:
@SamSamsonthe3rd please move non technical comments to the new channel on our discord!

comma commented:
```
code/modules/events/radiation_storm.dm:15:error: make_maint_all_access_torch: undefined proc
code/modules/events/radiation_storm.dm:54:error: revoke_maint_all_access_torch: undefined proc
code/modules/supermatter/supermatter.dm:283:error: unbolt_saferooms: undefined proc
```
My suggestion as to fix that is to move maint access procs into map datum and override on torch

HeyBanditoz committed:
Saferoom changes.

Radstorms, impending supermatter delam or red alert will now unbolt saferooms. Can now be bolted or unbolted via keycard auth. Bolt buttons moved inside. Everyone can now use the bolt buttons.
